airflow/providers/openlineage/provider.yaml

```diff
@@ -153,3 +153,10 @@ config:
         example: ~
         type: integer
         version_added: 1.9.0
+      include_full_task_info:
+        description: |
+          If true, OpenLineage event will include full task info - potentially containing large fields.
```

Comment:
Could you add an example of what "full task info" means? What all is included?

Reply:
Are there any high-level docs on what info can be gathered in the docs? and how users can control it apart from the config doc?

I was assuming something over here: https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/guides/structure.html but didn't find it

Reply:
@kaxil It's task dependent. I've attached two events, one with full task info enabled, one without, for S3ToGCSOperator: 
[af_with_full_info.json](https://github.com/user-attachments/files/16236609/af_with_full_info.json)
[af_without_full_task_info.json](https://github.com/user-attachments/files/16236610/af_without_full_task_info.json)


I've also added section to the user doc, where we explain particular conf variables: https://github.com/apache/airflow/pull/40589/files#diff-c1cb5b68b135a7ce9b52d925007cbbdcdaf03ec5e72871abcb1354561738b239

link to prod one: https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/guides/user.html